src/colorizer/TimeControls.ts

```diff
@@ -71,12 +67,18 @@ export default class TimeControls {
       const timeElapsed = endTime - startTime;
       onNewFrameCallback();
 
-      // TODO: Could add some sort of smoothing here to make the playback more consistent.
+      if (this.timerId === NO_TIMER_ID) {
```

Comment:
nit / DRY: all these conditional checks could now use isPlaying() :shrug:

Reply:
True! I sort of like having this here as a reminder that `isPlaying` relies on `this.timerId`... but it's probably cleaner to use `isPlaying` directly